Correct list of dashboards plugins (#10264)
opensearch-project/documentation-website: _install-and-configure/install-dashboards/plugins.md

| `mlCommonsDashboards` | [ml-commons-dashboards](https://github.com/opensearch-project/ml-commons-dashboards) | 2.6.0 |
| `notificationsDashboards` | [dashboards-notifications](https://github.com/opensearch-project/dashboards-notifications) | 2.0.0 |
| `observabilityDashboards` | [dashboards-observability](https://github.com/opensearch-project/dashboards-observability) | 2.0.0 |
| `queryInsightsDashboards` | [query-insights-dashboards](https://github.com/opensearch-project/query-insights-dashboards) | 2.19.0 |
| `queryWorkbenchDashboards` | [query-workbench](https://github.com/opensearch-project/dashboards-query-workbench) | 1.0.0 |
| `reportsDashboards` | [dashboards-reporting](https://github.com/opensearch-project/dashboards-reporting) | 1.0.0 |
| `searchRelevanceDashboards` | [dashboards-search-relevance](https://github.com/opensearch-project/dashboards-search-relevance) | 2.4.0 |
| `securityAnalyticsDashboards` | [security-analytics-dashboards-plugin](https://github.com/opensearch-project/security-analytics-dashboards-plugin)| 2.4.0 |
| `securityDashboards` | [security-dashboards-plugin](https://github.com/opensearch-project/security-dashboards-plugin) | 1.0.0 |

> _<sup>*</sup>`dashboardNotebooks` was merged in to the Observability plugin with the release of OpenSearch 1.2.0._<br>

Reviewer: ```suggestion
_<sup>*</sup>`dashboardNotebooks` was merged into the Observability plugin with the release of OpenSearch 1.2.0._<br>
```

```bash
sudo bin/opensearch-dashboards-plugin install alertingDashboards
```
{% include copy.html %}

## Viewing a list of installed plugins

To view the list of installed plugins from the command line, use the following command:

```bash
sudo bin/opensearch-dashboards-plugin list
```
{% include copy.html %}

The command returns the list of installed plugins and their versions:

```bash
alertingDashboards@3.1.0.0
anomalyDetectionDashboards@3.1.0.0
assistantDashboards@3.1.0.0
customImportMapDashboards@3.1.0.0
flowFrameworkDashboards@3.1.0.0
indexManagementDashboards@3.1.0.0
mlCommonsDashboards@3.1.0.0
notificationsDashboards@3.1.0.0
observabilityDashboards@3.1.0.0
queryInsightsDashboards@3.1.0.0
queryWorkbenchDashboards@3.1.0.0
reportsDashboards@3.1.0.0
searchRelevanceDashboards@3.1.0.0
securityAnalyticsDashboards@3.1.0.0
```

## Remove plugins

To remove a plugin, use the following command:

```bash
sudo bin/opensearch-dashboards-plugin remove alertingPlugin
```
{% include copy.html %}

Then remove all associated entries from `opensearch_dashboards.yml`.

> For certain plugins, you must also remove the optimized bundle. This is a sample command for the Anomaly Detection plugin:

Author: @peterzhuamazon Could you clarify what plugins require this step so users know?

Reviewer 2: https://github.com/opensearch-project/opensearch-build/blob/main/manifests/3.1.0/opensearch-dashboards-3.1.0.yml

Reviewer 2: On a glance this is correct compares to the manifest, since OSD core + FTrepo is not plugins and the list is complete.

> For certain plugins, you must also remove the optimized bundle. This is a sample command for the Anomaly Detection plugin:

Reviewer 2: ```suggestion

```

```bash
sudo bin/opensearch-dashboards-plugin list
```
{% include copy.html %}

The command returns the list of installed plugins and their versions:

```bash
alertingDashboards@3.1.0.0
anomalyDetectionDashboards@3.1.0.0
assistantDashboards@3.1.0.0
customImportMapDashboards@3.1.0.0
flowFrameworkDashboards@3.1.0.0
indexManagementDashboards@3.1.0.0
mlCommonsDashboards@3.1.0.0
notificationsDashboards@3.1.0.0
observabilityDashboards@3.1.0.0
queryInsightsDashboards@3.1.0.0
queryWorkbenchDashboards@3.1.0.0
reportsDashboards@3.1.0.0
searchRelevanceDashboards@3.1.0.0
securityAnalyticsDashboards@3.1.0.0
```

## Removing a plugin

To remove a plugin, use the following command:

```bash
sudo bin/opensearch-dashboards-plugin remove alertingDashboards
```
{% include copy.html %}

Then remove all associated entries from `opensearch_dashboards.yml` and restart OpenSearch Dashboards. 

## Updating plugins

> OpenSearch Dashboards doesn’t update plugins. Instead, you must [remove the old version](#removing-a-plugin), [reinstall the plugin](#installing-a-plugin), and restart OpenSearch Dashboards.

Reviewer: ```suggestion
OpenSearch Dashboards doesn't update plugins. Instead, you must [remove the old version](#removing-a-plugin), [reinstall the plugin](#installing-a-plugin), and restart OpenSearch Dashboards.
```

   ```bash
   sudo bin/opensearch-dashboards-plugin remove <plugin-name>
   ```

>    {% include copy.html %}
> 
> 1. Remove the optimized bundle:
> 
>    ```bash
>    sudo rm /usr/share/opensearch-dashboards/optimize/bundles/<bundle-name>
>    ```

Reviewer 2: ```suggestion
```
